app/templates/jinja2/email/signup.md

```diff
@@ -32,7 +32,4 @@ tweets texted to you by texting "follow @HVZAlertSystem" to the number
 21212.
 
 If you need to rent a blaster from the Humans vs Zombie Society, you can
-email us at uwhumansvszombies\@gmail.com for more information.
-
-[**You can find all rule changes and additions
-here.**](https://docs.google.com/document/d/1qWEV0cvqfqg9o8IvreCV2yV52SDmmdyV562-XO9Hl84/edit)
+email us at uwhumansvszombies\@gmail.com for more information.
```

Comment:
Curious as to why this is being removed. Has it been decided that that google doc is no longer the source of truth to rule changes/additions?